fixtures/gravity/index.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+import "babel-polyfill"
+
+import { spawnSync } from "child_process"
+import { basename, dirname, extname, join } from "path"
+import { writeFile } from "fs"
+
+import request from "request-promise"
```

Comment:
FWIW I've found [superagent](https://github.com/visionmedia/superagent) to have a nicer API and has promises built in. #superminor